internal/zinc-core/src/main/scala/sbt/internal/inc/IncrementalCommon.scala

```diff
@@ -85,25 +85,22 @@ private[inc] abstract class IncrementalCommon(log: sbt.util.Logger, options: Inc
     apis: APIs): Set[String]
 
   /**
-   * Logs API changes using debug-level logging. The API are obtained using the APIDiff class.
-   *
-   * NOTE: This method creates a new APIDiff instance on every invocation.
+   * Logs API changes using debug-level logging.
    */
   private def logApiChanges(apiChanges: Iterable[APIChange], oldAPIMapping: String => AnalyzedClass,
     newAPIMapping: String => AnalyzedClass): Unit = {
     val contextSize = options.apiDiffContextSize
     try {
       val wrappedLog = new Incremental.PrefixingLogger("[diff] ")(log)
-      val apiDiff = new APIDiff
       apiChanges foreach {
         case APIChangeDueToMacroDefinition(src) =>
           wrappedLog.debug(s"Public API is considered to be changed because $src contains a macro definition.")
         case apiChange: NamesChange =>
           val src = apiChange.modifiedClass
           val oldApi = oldAPIMapping(src)
           val newApi = newAPIMapping(src)
-          val apiUnifiedPatch = apiDiff.generateApiDiff(src.toString, oldApi.api, newApi.api, contextSize)
-          wrappedLog.debug(s"Detected a change in a public API ($src):\n$apiUnifiedPatch")
+          wrappedLog.debug(s"""Detected a change in a public API ($src):
+                              |old hash = ${oldApi.apiHash}, new hash = ${newApi.apiHash}""".stripMargin)
```

Comment:
Wait, does that mean that API diffs are no longer printed? They are *extremely* useful when debugging/developing incremental compilation